ee/cis/win-10/cis-policy-queries-WIP.yml

```diff
@@ -0,0 +1,127 @@
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Allow Use of Camera' is set to 'Disabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting controls whether the use of Camera devices on the machine are
+permitted.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to Disabled:
+    'Computer Configuration\Policies\Administrative Templates\Windows Components\Camera\Allow Use of Camera'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Camera\AllowCamera' AND data = 0);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level2, CIS_win10_enterprise_1.12.0, CIS_bullet_18.9.
+  contributors: rachelelysia
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Turn off cloud optimized content' is set to 'Enabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting turns off cloud optimized content in all Windows experiences.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to Enabled:
+    'Computer Configuration\Policies\Administrative Templates\Windows Components\Cloud Content\Turn off cloud optimized content'
+  query: |
+    Select 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Windows\\CloudContent\DisableCloudOptimizedContent' AND data = 1);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_18.9.14.2
+  contributors: rachelelysia
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Turn off Microsoft consumer experiences' is set to 'Enabled'
+  platforms: win10
+  platform: windows
+  description: |
+    This policy setting turns off experiences that help consumers make the most of their devices and Microsoft account.
+  resolution: |
+    To establish the recommended configuration via GP, set the following UI path to Enabled:
+    'Computer Configuration\Policies\Administrative Templates\Windows Components\Cloud Content\Turn off Microsoft consumer experiences'
+  query: |
+    SELECT 1 FROM registry WHERE (path = 'HKEY_LOCAL_MACHINE\\SOFTWARE\\Policies\\Microsoft\\Windows\\CloudContent\DisableWindowsConsumerFeatures' AND data = 1);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS_win10_enterprise_1.12.0, CIS_bullet_18.9.14.3
+  contributors: rachelelysia
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: >
+    CIS - Ensure 'Require pin for pairing' is set to 'Enabled: First Time' OR 'Enabled: Always'
```

Comment:
is the `:` on the sentence `'Enabled: Always'` not breaking the yml here?

Reply:
No, because I used a `>` to put the `name` value on the next line. Just looked it up and it's called a folded block scalar which allows you to write the string on the next line.

```
  name: >
    CIS - Ensure 'Require pin for pairing' is set to 'Enabled: First Time' OR 'Enabled: Always'
 ```

Reply:
ahh, that's right! thanks for mentioning it